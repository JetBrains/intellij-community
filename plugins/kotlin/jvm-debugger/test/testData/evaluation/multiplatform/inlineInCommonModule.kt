// MODULE: common
// FILE: common.kt

inline fun foo(): Int = 42

fun boo() {
    "".toString()
}

// MODULE: jvm(common)
// FILE: jvm.kt

fun main() {
    //Breakpoint!
    boo()
}

// STEP_INTO: 1
// EXPRESSION: foo()
// RESULT: 42: I

// IGNORE_K2
