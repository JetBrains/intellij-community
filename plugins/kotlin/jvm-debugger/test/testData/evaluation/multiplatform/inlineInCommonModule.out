LineBreakpoint created at jvm.kt:6
Run Java
Connected to the target VM
jvm.kt:6
Compile bytecode for foo()
Disconnected from the target VM

Process finished with exit code 0
